maven-model-builder/src/test/java/org/apache/maven/model/validation/DefaultModelValidatorTest.java

```diff
@@ -411,14 +411,19 @@ public void testIncompleteParent()
         assertTrue( result.getFatals().get( 2 ).contains( "parent.version" ) );
     }
 
-    public void testHardCodedSystemPath()
+    public void testHardCodedSystemPathAndDeprecatedScope()
```

Comment:
I think it's better having different test cases. Keeping the old test cases and create new ones for the changed behaviour. That makes it safe not to break previous behaviour,